src/EnergyPlus/WindowManager.cc

```diff
@@ -6977,146 +7257,17 @@ namespace WindowManager {
 
             if (iter >= 1) state.dataWindowManager->hcin = 0.5 * (hcinprev + state.dataWindowManager->hcin);
 
+            state.dataWindowManager->hcin *= adjRatio;
```

Comment:
I find it curious that this adjustment gets applied  after the damping in the line above. Seems like that would slow down convergence.

Reply:
It would, but It has to be adjusted after reevaluating `hcin` to be consistent with the "wetted area adjustment" assumption, otherwise, the adjustment would be overwritten. However, the `wettedAreaAdjRatio` would only be larger than 1.0 with high U input (> 5.7).